features/support/env.rb

```diff
@@ -14,6 +15,8 @@
 #
 # Chromedriver.set_version "2.24" unless ENV.has_key?('SEMAPHORECI')
 
+Webdrivers.install_dir = Rails.root.join('features', 'support', 'webdrivers')
+
 ActionController::Base.allow_rescue = false
 
```

Comment:
I actually just found a problem with this:
1. Need to call Webdrivers::Chrome.update in the `Before` block so that it is updated before any scenarios are run.
   - this may trigger a web request (if an update is done), which means that both VCR and WebMock must be cconfigured.

So the configuration needs to be moved to this file.  Here's what worked for me  (I put this right before the `Before` block:

```

# -------------------------
# Configure WebMock and VCR. Have to have them always configured (for all scenarios)
# in case Webdrivers needs to update the driver, which will force a request.
#
# Do not use VCR or WebMock on sites that Webdrivers gem uses to download and update drivers:
VCR.configure do |c|
  c.ignore_hosts('chromedriver.storage.googleapis.com')
end

webdrivers_download_sites = [
    "https://chromedriver.storage.googleapis.com",
    "https://github.com/mozilla/geckodriver/releases",
    "https://selenium-release.storage.googleapis.com",
    "https://developer.microsoft.com/en-us/microsoft-edge/tools/webdriver"
]
WebMock.disable_net_connect!(allow_localhost: true, allow: webdrivers_download_sites)

VCR.configure do |c|
  c.hook_into :webmock
  c.cassette_library_dir     = 'features/vcr_cassettes'
  c.allow_http_connections_when_no_cassette = true
  c.ignore_localhost = true
  c.default_cassette_options = { allow_playback_repeats: true }
  c.ignore_hosts('chromedriver.storage.googleapis.com')

  webdrivers_download_sites.each do | webdriver_download_site |
    c.ignore_hosts(webdriver_download_site)
  end
end
# -------------------------

```

Reply:
@weedySeaDragon - what problems were you seeing?  My tests all pass locally as well as in CI (aside from an occasional flaky test failure).

Reply:
I don't see where you're calling `Webdrivers::Chrome.update`

When I put `Webdrivers::Chrome.update` in to `env.rb` I started getting errors -- which is what lead me to having to configure WebMock and VCR in `env.rb` as above.

Reply:
The update check occurs when a web driver is invoked (in a test) from what I've seen.  The only place where this was affecting out tests was in CI, when an attempted update failed due to VCR not recognizing the outgoing request.  I updated the appropriate hook in hooks.rb to fix this.

The update occurs every time in our current version, but will go to a 24 hour interval on version 4.x (https://github.com/titusfortner/webdrivers#caching-drivers).

Reply:
```Before('@dinkurs_fetch or @dinkurs_invalid_key') do
  VCR.configure do |c|
    c.hook_into :webmock
    c.cassette_library_dir     = 'features/vcr_cassettes'
    c.allow_http_connections_when_no_cassette = true
    c.ignore_localhost = true
    c.default_cassette_options = { allow_playback_repeats: true }
    c.ignore_hosts('chromedriver.storage.googleapis.com')
  end
```